spec/factories/company_values.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :company_value do
+    sequence(:title) { |i| "sample title #{i}" }
```

Comment:
consider using more descriptive names - it will be easier to find it on the page during tests, and allow to avoid collisions with other objects with 'sample title x' :) 
```suggestion
    sequence(:title) { |i| "Company Value title #{i}" }
```